### ISSUE TITLE
running from repo as working directory fails

### ISSUE THREAD

**Issue by bertsky** [poke1024/pyalign#5]
If I `import pyalign` while still inside the source directory, I get the following:
```
RuntimeError: none of ['pyalign.algorithm.native', 'pyalign.algorithm.intel_avx2', 'pyalign.algorithm.apple_m1', 'pyalign.algorithm.generic'] is available
```

(The problem goes away when `cd`ing somewhere else.)

**Comment by poke1024** [comment]
Yes, I know, the problem here is that the git repo contains a folder called `pyalign` and `import pyalign` tries to import that folder, instead of the module. Maybe I should rename that folder to `pyalign-src` or something similar.
